Content.Server/Forensics/Systems/ForensicPadSystem.cs

```diff
@@ -80,8 +80,10 @@ private void OnAfterInteract(EntityUid uid, ForensicPadComponent component, Afte
                 return;
             }
 
-            if (TryComp<FiberComponent>(args.Target, out var fiber))
-                StartScan(uid, args.User, args.Target.Value, component, string.IsNullOrEmpty(fiber.FiberColor) ? Loc.GetString("forensic-fibers", ("material", fiber.FiberMaterial)) : Loc.GetString("forensic-fibers-colored", ("color", fiber.FiberColor), ("material", fiber.FiberMaterial)));
+            // Begin DeltaV changes - add Fiberprint to display
+            if (TryComp<FiberComponent>(args.Target, out var fiber) && fiber.Fiberprint != null)
```

Comment:
Now it won't display anything if the fiber doesn't have a print, is this intentional?
Otherwise try and check for null in the loc